### PR TITLE
Switch to static tab

### PR DESCRIPTION
**Pull request by paulstn**
The ability for a hotkey (CTRL+Q) to switch to the last tab, meant to be a prototype that has the core functionality without a MRU cache implemented

Fixed Issues:
- Having our own functionality to change shortcut keys is not needed, there is already a page to allow for that: chrome://extensions/shortcuts. However, we could still add in our own page to do that - within the popup perhaps? [issue](https://github.com/paulstn/SmarterTabSwitching/issues/12)

TODO in another PR:
- Fix all known issues below
- Convert 'changing by tab index' logic over to 'switching by tab id'

Known Issues: 
- [TAB] is a protected key by Chrome, we can later decide if we want the default to be some other key or try to implement ways to still override CTRL+TAB from Chrome. [issue](https://github.com/paulstn/SmarterTabSwitching/issues/25)
- Switching to tabs when the order of tabs change and/or number of tabs change needs to be fixed. [issue](https://github.com/paulstn/SmarterTabSwitching/issues/23)
  - One solution: Use a comprehensive cache that is a stack of tab ids - ids are unique (handles moving tabs) and the stack will present whatever tab is on top, if tabs need to be removed from the stack (handles deleted tabs). Will need logic to figure out where a tab's index is, or maybe even later cache indices for better performance. 
- Initialization is wrong. Instead of setting lastTabIndex to 0, it should be set to an actual tab's index. [issue](https://github.com/paulstn/SmarterTabSwitching/issues/24)
- Tab Switching does not work well with multiple windows, seems to break easily. [issue](https://github.com/paulstn/SmarterTabSwitching/issues/22)

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -2,10 +2,21 @@
...
+    "service_worker": "background.js"
+  },
+  "action": {
+    "default_title": "Click to show an alert",
+    "default_popup": "popup.html"
+  },
+  "commands": {
+    "switch-tab": {
+      "suggested_key": {
+        "default": "Ctrl+Q"
```
Comment (Soreloser2) [inline src/manifest.json:16]: Does this mean that the keybind by default will be Ctrl + Q?

**Review by Soreloser2 — APPROVED** [review]

File: src/manifest.json (hunk 1 above)
Comment (paulstn) [reply src/manifest.json:16]: For now. We could choose another default, if you think some other keybind would be better. Or, like I say in this issue: https://github.com/paulstn/SmarterTabSwitching/issues/25, we could try to make CTRL+TAB work, but thats much much later on the list of goals. We could replace one of our 'stretch goals' with this goal instead.

Comment (Soreloser2) [reply src/manifest.json:16]: Q sounds good to me

**Review by kashish-ops — COMMENTED** [review]
Reviewed background.js and manifest.json

**Review by kashish-ops — APPROVED** [review]

**Pull request merged**